src/Didomi.ts

```diff
@@ -599,6 +603,28 @@ export const Didomi = {
     }
   },
 
+  /**
+   *  Set user information with Encryption authentication
+   *
+   *  @param userAuthParams as UserAuthParams
+   *  @param synchronizedUsers as UserAuthParams[] (optional)
+   */
+  setUserWithAuthParams: (
+    userAuthParams: UserAuthParams,
+    synchronizedUsers?: UserAuthParams[] | undefined
+  ): void => RNDidomi.setUserWithAuthParams(JSON.stringify(userAuthParams), JSON.stringify(synchronizedUsers)),
+
+  /**
+   *  Set user information with Encryption authentication and check for missing consent
```

Comment:
if this function accepts both encryption and hashing we'll need to update this comment